eeps/eep-0007.md

```diff
@@ -255,11 +255,12 @@ following addition:
 The supported options are:
 
 - **`{preload, PreloadList}`**
+
   Preload the given list of functions, and prepare their
   call structures.  Each PreloadList element is a tuple
   in the form:
 
-      tuple(c_func_name(), `signature())
+        tuple(c_func_name(), signature())
```

Comment:
Was wrong. But my previous intention was to fix the Markdown, not the content of the HTML (which I may have ended up doing where I saw it, actually).